src/Meditation.UI/ViewModels/MainViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Avalonia;
+using Avalonia.Controls.ApplicationLifetimes;
+using CommunityToolkit.Mvvm.Input;
+using Meditation.UI.Windows;
+using System;
+
+namespace Meditation.UI.ViewModels;
+
+public partial class MainViewModel : ViewModelBase
+{
+    public string Greeting => "Welcome to Meditation!";
+
+    [RelayCommand]
+    public void DisplayAttachProcessWindow()
+    {
+        var applicationLifetime = (IClassicDesktopStyleApplicationLifetime?)(Application.Current?.ApplicationLifetime);
+        var ownerWindow = applicationLifetime?.MainWindow
+                          ?? throw new InvalidOperationException("Can not open modal, as parent window could not be resolved.");
```

Comment:
[ToFix] The word "cannot" is typically written as one word when it is used to indicate the inability to do something. For example: "I cannot attend the meeting."

It is written as two separate words, "can not," when "can" is used in its normal sense of being able to do something, and "not" is used to negate that ability. For example: "I can not only finish the project on time but also exceed expectations."

In practice, however, "cannot" is more commonly used as a single word in most contexts.